packages/transformer/src/test/TestUtils/TimelineTestUtil.ts

```diff
@@ -36,7 +36,11 @@ import {
 import { IModelTestUtils } from "./IModelTestUtils";
 import { omit } from "@itwin/core-bentley";
 
-const { saveAndPushChanges } = IModelTestUtils;
+const saveAndPushChanges = async (
```

Comment:
Previously was getting avoid referencing unbound methods which may cause unintentional scoping of 'this'. But then if I added this: void to show that its not using this
I would get a compiler error due to binding of saveAndPushChanges to bind accessToken arg to all calls, on line 131 in hub.test.ts. So I went with this solution, resolve if fine with that.